Makefile

```diff
@@ -234,3 +234,6 @@ shake_concourse_volumes: check-env ## Restarts concourse services and workers an
 show-cf-memory-usage: ## Show the memory usage of the current CF cluster
 	$(eval export API_ENDPOINT=https://api.${SYSTEM_DNS_ZONE_NAME})
 	@./scripts/show-cf-memory-usage.rb
+
+fix_sso_url: check-env
```

Comment:
It needs `SYSTEM_DNS_ZONE_NAME` and `check-env` is currently our only way of (indirectly) checking whether one of the environment targets have been run.